_P222_DebouncedCounter.ino

```diff
@@ -22,13 +22,15 @@ void Plugin_222_pulse_interrupt4() ICACHE_RAM_ATTR;
 
 unsigned long Plugin_222_pulseCounter[TASKS_MAX];
 unsigned long Plugin_222_pulseTotalCounter[TASKS_MAX];
-//unsigned long Plugin_222_pulseTime[TASKS_MAX];
-//unsigned long Plugin_222_pulseTimePrevious[TASKS_MAX];
+unsigned long Plugin_222_pulseTime[TASKS_MAX];
+unsigned long Plugin_222_pulseTimePrevious[TASKS_MAX];
 byte Plugin_222_pulsePin[TASKS_MAX];
 byte Plugin_222_pulseMode[TASKS_MAX];
 byte Plugin_222_ct0[TASKS_MAX];
 byte Plugin_222_ct1[TASKS_MAX];
-byte Plugin_222_key_state[TASKS_MAX];
+byte Plugin_222_prescaler[TASKS_MAX];
+byte Plugin_222_prescaler_cnt[TASKS_MAX];
+byte Plugin_222_signal_state[TASKS_MAX];
 byte Plugin_222_signal_falling[TASKS_MAX];
 byte Plugin_222_signal_rising[TASKS_MAX];
```

Comment:
This does allocate `TASK_MAX` * (9 bytes + 4 longs = 25 bytes) = 300 bytes.
That's a lot of memory allocated on sys stack regardless whether this is being used.